src/assets/variables.scss

```diff
@@ -1,20 +1,34 @@
 /*colors*/
 //blue
-$primary001: #7BB9FA;
-$primary002: #499FF8;
+$primary001: #7bb9fa;
+$primary002: #499ff8;
 $primary003: #d7eafd;
 //green
-$success001: #38DB81;
-$success002: #21B965;
-$success003: #86E9B2;
+$success001: #38db81;
+$success002: #21b965;
+$success003: #86e9b2;
 //red
-$danger001: #F67F7F;
-$danger002: #F34E4E;
-$danger003: #FCD9D9;
+$danger001: #f67f7f;
+$danger002: #f34e4e;
+$danger003: #fcd9d9;
 //grey
-$grey001: #D0D0D0;
-$grey002: #BFBFBF;
-$grey003: #F1F1F1;
+$grey001: #d0d0d0;
+$grey002: #bfbfbf;
+$grey003: #f1f1f1;
 //other
 $black: #282828;
-$white: #FFFFFF;
\ No newline at end of file
+$white: #ffffff;
+
+/*typography*/
+//font
+@import url("https://fonts.googleapis.com/css?family=Montserrat");
+// font size
+$h1: 48px;
+$h2: 34px;
+$h3: 28px;
+$h4: 18px;
+$paragraph: 14px;
+// font weight
+$regular: 500;
```

Comment:
I do have second thoughts after checking the google fonts and I'm wondering if we shouldn't go with 
```
$regular: 400;
$semibold: 600;
$semibold: 800;
```